clients/drcachesim/tools/opcode_mix.h

```diff
@@ -44,6 +44,7 @@
 
 #include "dr_api.h" // Must be before trace_entry.h from analysis_tool.h.
 #include "analysis_tool.h"
+#include "decode_cache.h"
```

Comment:
It is probably also worth adding a section to drcachesim.dox.in about this new library since this is of general interest to multiple analyzers.  Put a xref in the format section where today it briefly talks about using drdecode.

Reply:
Done